src/client/src/components/AddTaskForm/AddTaskForm.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import ItemClient from "../../services/taskService";
+
+import "./addTaskForm.css";
+
+const AddTaskForm = ({
+  tasks,
+  setTasks,
+  editTask,
+  setEditTask,
+  setErrorMessage,
+  setIsErrorShown,
+}) => {
+  const [input, setInput] = useState("");
+
+  const taskService = new ItemClient();
```

Comment:
same comment about re-creating the client on each re-render, this could be wrapped with useMemo and create this object only on mount